skins/larry/lastlogin.css

```diff
@@ -1,6 +1,7 @@
 .lastlogin TABLE {
 	border-collapse:collapse;
-	border:1px solid #bbd3da;
+	border:1px solid #1b1b1b;
```

Comment:
Hi,
Why did you change the border color? Any specific reason?

Reply:
Oh, must have done it by mistake, That was for another project of mine.